rich/pretty.py

```diff
@@ -63,7 +63,7 @@ def install(
 
     console = console or get_console()
 
-    def display_hook(value: Any) -> None:
+    def display_hook(value: Any, *_args, **_kwargs) -> None:
```

Comment:
Is this for iPython's benefit? sys.displayhook doesn't take any additional argument according to docs.

Reply:
It was there to allow it to work with the ipython display hook, but is no longer needed